### PR TITLE
Fixing to match the current Mathics version

### PR DESCRIPTION
**Pull request by mmatera**
This PR fix errors comming from the recent change in the Mathics core interface: 
* LineFeeder and MultilineFeeder are now called MathicsLineFeeder and MathicsMultilineFeeder
* Some changes in the way that docstrings are reader requires an update in mathics_djando doc module.

### CODE REVIEW

File: mathics_django/web/views.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def query(request):
         query_log.result = str(result)  # evaluation.results
         query_log.error = False
         query_log.save()
-
+    print(result)
```
Comment (rocky) [inline mathics_django/web/views.py:163]: Probably don't want print here.

**Review by rocky — COMMENTED** [review]
I'll merge this into rc1 now with small changes and it can get merged into master from there.

File: mathics_django/web/views.py (hunk 1 above)
Comment (mmatera) [reply mathics_django/web/views.py:163]: sure...

Comment (mmatera) [reply mathics_django/web/views.py]: Done.

**Comment by rocky** [comment]
Has been merged via rc1 into master.

**Pull request closed without merging**